### PR TITLE
Add two more DNS Settings deprecations

### PR DESCRIPTION
**Pull request by ghost**
### Summary

See diff.

### Screenshots (optional)

None.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [x] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [x] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

cc: @RebeccaTamachiro, @hannes-cf

### CODE REVIEW

File: src/content/changelogs/api-deprecations.yaml [hunk 1]
```diff
@@ -5,6 +5,47 @@ productLink: "/fundamentals/"
...
+    - `PATCH /zones/:zone_id/settings/cname_flattening`
+
+- publish_date: '2025-03-14'
+  title: "Account Settings: default_nameservers and use_account_custom_ns_by_default"
+  description: |-
+    Deprecation date: March 14, 2025
+
+    The fields `"default_nameservers"` and `"use_account_custom_ns_by_default"` within the `"settings"` object of accounts are deprecated.
+    Instead, use the [Show DNS Settings](/api/operations/dns-settings-for-an-account-list-dns-settings) and [Update DNS Settings](/api/operations/dns-settings-for-an-account-update-dns-settings) endpoints to manage this setting.
+    This setting is available in the new API as `.zone_defaults.nameservers.type`, with allowed values `"cloudflare.standard"`, `"custom.account"` and `"custom.tenant"`.
```
Comment (hannes-cf) [inline src/content/changelogs/api-deprecations.yaml]: Why don't we mention `cloudflare.standard.random` and `cloudflare.foundation` here?

Comment (ghost) [reply src/content/changelogs/api-deprecations.yaml]: `cloudflare.foundation` is a separate setting. `cloudflare.standard.random` isn't available in the deprecated API but good point, probably wouldn't hurt to list it nevertheless.

**Comment by RebeccaTamachiro** [comment]
Moving to #16972 as I got some weird issues with git and couldn't push conflict and indentation fixes to this PR

**Comment by ghost** [comment]
Thanks Rebecca! :pray:

**Pull request closed without merging**